optaplanner-website-root/content/blog/2023/04/24/OptaPlanner-9-released.adoc

```diff
@@ -0,0 +1,97 @@
+= OptaPlanner 9 has been released
+rsynek
+2023-04-24
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: migration, jakarta, javax, javaee, quarkus, spring boot
+:jbake-social_media_share_image: migration-flowchart.png
+
+In the previous article link:../../02/21/OptaPlanner-9-is-coming.adoc[OptaPlanner 9 is coming] we explained why we are doing OptaPlanner 9.
+Now OptaPlanner 9.37.0.Final has been released to enable users migrate to Jakarta EE, Quarkus 3 and Spring Boot 3.
+
+== Differences between OptaPlanner 8 and 9
+
+OptaPlanner 8 and 9 are fully compatible regarding their APIs and equal feature-wise.
```

Comment:
This is not technically true, right? The Hibernate change?

Reply:
Indeed, thanks for the reminder.